docs/specifications/category-3d-tiles.md

```diff
@@ -101,3 +95,27 @@ Following vis.gl conventions, `attributes` are represented by "glTF-style" acces
 | `size`           | No    | `Number`     | Number of components, `1`-`4`.                                                                                               |
 | `byteOffset`     | Yes   | `Number`     | Starting offset into the bufferView.                                                                                         |
 | `count`          | Yes   | `Number`     | The number of elements/vertices in the attribute data.                                                                       |
+
+
+## Helper Classes
+
+Tileset Traversal Support
+
+To start loading tiles once a top-level tileset file is loaded, the application can instantiate the `Tileset3D` class and start calling `tileset3D.traverse(camera_parameters)`.
+
```

Comment:
Provide more details about `camera_parameters`, also is it the appropriate name for this parameter?

Reply:
Yes. This PR was mainly intended to reorganize the existing text so that the category docs follow the same structure (and also does some cleanup of the image category).

I should do a separate PR with another pass on the 3D tiles category docs.